TmdbEasy.Tests/V3TmdbEasyClientTests.cs

```diff
@@ -23,63 +22,5 @@ public void NullOptions_Throws_ArgumentNullException()
 
             Assert.Throws<ArgumentNullException>(() => new TmdbEasyClientv3(_jsonDeserializer, options));
         }
-
-        [Test]
-        public void OptionsWithSsl_Off_SetsCorrectVersionOfUrl_NoSsl()
-        {
-            var options = new TmdbEasyOptions()
-            {
-                UseSsl = false
-            };
-
-            var client = new TmdbEasyClientv3(_jsonDeserializer, options);
-
-            Assert.AreEqual(UrlConstants.TmdbUrl3, client.GetBaseUrl());
-        }
-
-        [Test]
-        public void OptionsWithSsl_On_SetsCorrectVersionOfUrl_WithSsl()
-        {
-            var options = new TmdbEasyOptions()
-            {
-                UseSsl = true
-            };
-
-            var client = new TmdbEasyClientv3(_jsonDeserializer, options);
-
-            Assert.AreEqual(UrlConstants.TmdbUrl3Ssl, client.GetBaseUrl());
-        }
-
-        [Test]
-        [TestCase("asdjhfbakjsdhb")]
-        [TestCase("")]
-        [TestCase(null)]
-        public void OptionsWithApiKey_SetsAndGetsApiKey(string apikey)
-        {
-            var options = new TmdbEasyOptions()
-            {
-                ApiKey = apikey
```

Comment:
Now that API key not accessable i can say that the live API calls are good enough test cases that the API key was set correctly (no 401 HTTP status codes).

Reply:
I guess we can assume that